packages/dx-react-demos/src/demo-sources/chart-basic/bootstrap3/area.skip.jsx

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable react/jsx-filename-extension */
+import * as React from 'react';
+import {
+  ArgumentAxis,
+  ValueAxis,
+  Grid,
+  Chart,
+  AreaSeries,
+} from '@devexpress/dx-react-chart-svg';
```

Comment:
Do you plan to merge this PR after #1046 ? If so, do not forget to change the `dx-react-chart-svg` package name.

Reply:
Yes, after merge it will be renamed.